tools/validator.py

```diff
@@ -80,14 +80,18 @@ def CheckFile(self, filename):
         for source in artifact_definition.sources:
           if source.type_indicator in (
               definitions.TYPE_INDICATOR_WINDOWS_REGISTRY_KEY):
-            if self._HasDuplicateRegistryKeyPaths(
-                filename, artifact_definition, source):
+
+            # Exempt the legacy file from duplicate checking because it has
+            # duplicates intentionally.
+            if (filename != 'definitions/legacy.yaml' and
+                self._HasDuplicateRegistryKeyPaths(filename,
```

Comment:
I use the auto python formatter since we started using it in grr. My editor is set to format on save, never have to fix a lint error again:
https://github.com/google/yapf

There's occasions when I would format differently, like this one. But the advantage of having it done automatically is greater I think.